docs/application/vscode-ext/index.md

```diff
@@ -14,7 +14,7 @@ Visual Studio Code Extension for Tizen enables you to develop Tizen .NET and Tiz
 
   To use VS Code Extension for Tizen, you must install the following tools:
 
-  - NET Core SDK 2.0 or later **for Tizen.Net only**
+  - NET Core SDK 2.0 **for Tizen.Net only**
```

Comment:
```suggestion
  - NET Core SDK 2.0 **for Tizen .Net only**
```

Please see the Tizen naming rules: https://github.com/Samsung/tizen-docs/blob/master/styleguide/naming-rules.md.